Projects/hCoreUI/Sources/hForm/hForm.swift

```diff
@@ -110,26 +140,24 @@ public struct hForm<Content: View>: View {
                     }
                 }
             } else {
-                BackgroundView().edgesIgnoringSafeArea(.all)
-            }
-            ScrollView {
-                VStack {
-                    if let hFormTitle, hUseNewStyle{
-                        Text(hFormTitle.1)
-                            .padding(.top, hFormTitle.0.topMargin)
-                            .padding(.bottom, hFormTitle.0.bottomMargin)
-                    }
-                    content
+                if hUseBlur {
+                    BackgroundBlurView().edgesIgnoringSafeArea(.all)
+                } else {
+                    BackgroundView().edgesIgnoringSafeArea(.all)
                 }
-                .frame(maxWidth: .infinity)
-                .tint(hForm<Content>.returnTintColor(useNewStyle: hUseNewStyle))
-                Color.clear
-                    .frame(height: bottomAttachedViewHeight)
             }
-            .modifier(ForceScrollViewIndicatorInset(insetBottom: bottomAttachedViewHeight))
-            .findScrollView { scrollView in
-                if #available(iOS 15, *) {
-                    scrollView.viewController?.setContentScrollView(scrollView)
+            if hDisableScroll {
```

Comment:
Instaed of having different content depending on hDisableScroll (since we are using scrollView when we are using .detent style) i suggest to do it like this:
            .findScrollView { scrollView in
                if #available(iOS 15, *) {
                    scrollView.viewController?.setContentScrollView(scrollView)
                }
                if hDisableScroll {
                    scrollView.addGestureRecognizer(UIPanGestureRecognizer())
                }
            }